app/uk/gov/hmrc/gform/controllers/DeclarationController.scala

```diff
@@ -73,12 +74,12 @@ class DeclarationController @Inject() (
               response <- gformConnector.submitForm(formId)
               template <- gformConnector.getFormTemplate(theForm.formTemplateId)
               _ <- repeatService.clearSession
+              ackPage <- acknowledgementPage(template, formId)
             } yield {
               auditService.sendSubmissionEvent(theForm, formTemplate.sections)
               Ok(Json.obj("envelope" -> response.body, "formId" -> Json.toJson(formId)))
```

Comment:
Looks like `Ok(Json.obj("envelope" -> response.body, "formId" -> Json.toJson(formId)))` isn't needed anymore. Could we remove it?